build/verify-manifests.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash -ex
+
+TAG_CURRENT="${1}"
+BUNDLE_DIR="${2:-manifests/node-maintenance-operator}"
+VERIFY_DIR="manifests/verify-dir"
+
+function compare_files {
+	local file_a="$1"
+	local file_b="$2"
+
+	set +e
+	diff "$file_a" "$file_b"
+	if [[ $? != 0 ]]; then
+		echo "$file_a differ from $file_b"
+		cat "$file_b"
```

Comment:
I suggest to output the file path instead of its content, or at least swap lines 14 and 15

Reply:
I added this because update-olm.py worked differently on my machine as compared with the build machine. (very scary, but I had to address these issues and see what it is doing, so I needed the file content).

Reply:
@MoserMichael please resolve the conversation if you have addressed the feedback.  Otherwise it looks like the PR is not ready to be merged.